### PR TITLE
DOCS-765: Remove /instance refs and update example files link

### PR DESCRIPTION
**Pull request by oliverhowell**

### CODE REVIEW

**Review by frant-hartm — APPROVED** [review]

File: docs/modules/configuration/pages/jet-configuration.adoc [hunk 1]
```diff
@@ -48,7 +48,7 @@ HazelcastInstance instance = createHazelcastInstance(config);
 ====
 
 Note that, Hazelcast's Jet engine is disabled by default when you use Hazelcast
-within the Java context, i.e., `hazelcast.jar`. When using Hazelcast ZIP/TAR distributions
+within the Java context, i.e. `hazelcast.jar`. When using Hazelcast ZIP/TAR distributions
```
Comment (rebekah-lawrence) [inline docs/modules/configuration/pages/jet-configuration.adoc:51]:
```suggestion
within the Java context; that is, `hazelcast.jar`. When using Hazelcast ZIP/TAR distributions
```

File: docs/modules/configuration/pages/jet-configuration.adoc [hunk 2]
```diff
@@ -101,39 +101,36 @@ By default, resource uploading for Jet engine jobs is disabled.
 
 == List of Configuration Options
 
-The Hazelcast full example configuration files (https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/resources/hazelcast-full-example.yaml#L3509[YAML],
-https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/resources/hazelcast-full-example.xml#L3612[XML]) in your distribution
-includes a description of all configuration options.
-Jet engine configuration options are listed below:
+The _hazelcast-full-example_ configuration files (YAML, XML) available in your distribution and linked from xref:configuration:configuring-declaratively.adoc[] include a description of all configuration options. The configuration options for Jet engine are also listed below:
```
Comment (rebekah-lawrence) [inline docs/modules/configuration/pages/jet-configuration.adoc:104]: can we clarify that it isn't that both YAML and XML config files need to be updated?

Comment (rebekah-lawrence) [inline docs/modules/configuration/pages/jet-configuration.adoc:104]:
```suggestion
The _hazelcast-full-example_ configuration files (available as YAML or XML) available in your distribution and linked from xref:configuration:configuring-declaratively.adoc[] include a description of all configuration options. The configuration options for Jet engine are also listed below:
```

File: docs/modules/configuration/pages/jet-configuration.adoc [hunk 3]
```diff
@@ -319,7 +313,7 @@ JetService jetFromClient = client.getJet();
 Alternatively, you can add `hazelcast-client.yaml/xml` to the classpath or
 working directory which will be picked up automatically. The location of
 the file can also be given using the `hazelcast.client.config` system
-property, i.e., `-Dhazelcast.client.config=C:/myhazelcast-client.yaml/xml`.
+property, i.e. `-Dhazelcast.client.config=C:/myhazelcast-client.yaml/xml`.
```
Comment (rebekah-lawrence) [inline docs/modules/configuration/pages/jet-configuration.adoc:316]:
```suggestion
property; that is, `-Dhazelcast.client.config=C:/myhazelcast-client.yaml/xml`.
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/configuration/pages/jet-configuration.adoc (hunk 2 above)
Comment (oliverhowell) [reply docs/modules/configuration/pages/jet-configuration.adoc:104]: Not sure what you mean here - the files are included just for reference. For context I changed the hardcoded link to these files to point to the page instead - are you meaning we should provide more context on how these files are used?

Comment (oliverhowell) [reply docs/modules/configuration/pages/jet-configuration.adoc]: Op, took your suggestion - think that clarifies

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Comment by oliverhowell** [comment]
What versions should I backport this to?

**Pull request merged**